docs/canonicalk8s/snap/howto/index.md

```diff
@@ -57,6 +57,7 @@ security/index
 :titlesonly:
 Manage upgrades <upgrades>
 Refresh Kubernetes Certificates <refresh-certs>
```

Comment:
We have three files related to certificates now - should we group them? Or put them under security?

Reply:
I think it should be better to move them to `Security`. Thanks for the suggestion!